crates/rpc-proto/src/lib.rs

```diff
@@ -3,8 +3,7 @@
 #[cfg(feature = "std")]
 extern crate std;
 
-mod proto_files;
-pub use proto_files::PROTO_FILES;
+pub use miden_node_proto::PROTO_FILES;
```

Comment:
Is exporting a `nostd` compliant `const` from an otherwise non-compliant dependency fine? I think it is -- haven't experimented much with `nostd`.